components/camera/replaypcd/replaypcd.go

```diff
@@ -81,12 +162,87 @@ func (replay *pcdCamera) Stream(ctx context.Context, errHandlers ...gostream.Err
 	return stream, errors.New("Stream is unimplemented")
 }
 
-// Close stops replay camera but is not implemented for replay.
+// Close stops replay camera and closes its connections to the cloud.
 func (replay *pcdCamera) Close(ctx context.Context) error {
-	return errors.New("Close is unimplemented")
+	replay.closeCloudConnection(ctx)
+	return nil
 }
 
 // Reconfigure will bring up a replay camera using the new config but is not implemented for replay.
-func (replay *pcdCamera) Reconfigure(ctx context.Context, _ resource.Dependencies, cfg resource.Config) error {
-	return errors.New("Reconfigure is unimplemented")
+func (replay *pcdCamera) Reconfigure(ctx context.Context, deps resource.Dependencies, conf resource.Config) error {
```

Comment:
We need to prevent use of pcdCamera during reconfiguration, or if reconfiguration fails.

Reply:
Can this be handled by the new mutex lock and `closed` variable introduced thanks to your presentation on Thursday?

Reply:
yes, thank would work!